src/main/middleware/FirebaseAuthMiddleware.ts

```diff
@@ -6,7 +6,7 @@ import Auth = auth.Auth;
 /**
  * Middleware function to check if the request has correct client header using firebase-admin auth
  */
-export class FirebaseAuth {
+export class FirebaseAuthMiddleware {
```

Comment:
Renamed for more clarity because I got confused what the difference was between `FirebaseAuth` and `Auth`